cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -26,20 +25,10 @@
 ]
 
 
-@pytest.fixture(scope='module')
-def providers(request, second_provider, provider):
-    """ Fixture to setup providers """
-    setup_or_skip(request, second_provider)
-    setup_or_skip(request, provider)
-    yield second_provider, provider
-    second_provider.delete_if_exists(cancel=False)
-    provider.delete_if_exists(cancel=False)
-
-
 def _form_data_cluster_mapping(second_provider, provider):
```

Comment:
Seems to be doing the same thing as _form_data_datastore_mapping. It would be good to combine these into a single helper method. That would probably be out of scope in this PR, maybe a TODO?